docs/monorepo.md

```diff
@@ -86,6 +91,40 @@ To run one package's tests:
 
 `npm run test-packages [ test file pattern ]`
 
+## Building packages & apps
+
+Packages will have their `prepare` scripts run automatically on `npm install`.
+
+Independent apps in `/apps` directory use `build` instead of `prepare` so avoid unnecessary builds on `npm install`.
+
+You can build packages also by running:
+
+```bash
+npm run build-packages
+```
+
+Or even specific packages:
+
+```bash
+npx lerna run prepare --scope="@automattic/calypso-build"
+```
+
+Or specific apps:
+
+```bash
+npx lerna run build --scope="@automattic/calypso-build"
+```
+
```

Comment:
Not sure if I should highlight `prepare`/`build` difference here and if I should explain why it exists?

Reply:
There is already something trying to clarifying that a few lines above: 

> Independent apps in `/apps` directory use `build` instead of `prepare` so avoid unnecessary builds on `npm install`.

But yeah, +1 to extend that and explain the differences between the 2 commands.

Reply:
Thanks, elaborated more in https://github.com/Automattic/wp-calypso/pull/32137/commits/69fbf250af639c58544b5aa7dc0e8e9465ea066f